tests/test_star.py

```diff
@@ -371,6 +371,10 @@ def test_star():
     assert star.flux == 1.
     assert star.center == (0,0)
     assert star.is_reserve == False
+    assert star['color_ri'] == 0.5
+    assert star['color_iz'] == -0.2
+    with np.testing.assert_raises(KeyError):
```

Comment:
When designing a unit test, how do you decide which exceptions to test for? It seems like there could be also be a `TypeError` when making a Star object, for example.

Reply:
At a minimum I try to include any errors that we explicitly raise in our code.  That doesn't apply in this case though.  This one was just because later I'll add color_gi using 'withProperties', so checking that it's not present here was just to add some parallelism to the two cases.